Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian:stable-slim
+FROM nvidia/cuda:latest
```

Comment:
Would you mind trying `10.2-base` or `10.2-runtime`? `latest` is 1.45 GB. You can probably remove the other nvidia packages and configuration you've added if you use one of those images.

Reply:
I can do some testing, but I don't think it'll work without applying the openCL packages manually. As it is I only get one GPU working not the 4 that the pod has access to.

Reply:
I started up a `g3.8xlarge`(2 GPUs) EC2 instance, used `FROM nvidia/cuda:9.2-base` in conjunction with the openCL configuration you've applied below, and was able to use both GPUs. I had issues using 10.2 with the Tesla M60, but 9.2 was fine.